### DISCUSSION TITLE
Kong admin prometheus metrics

### DISCUSSION THREAD

**Discussion by narenaryan** [Kong/kong#8632] [Help] [answered]
### Summary

We are using Kong open source plugin installed in DB-less mode via Helm. We also enabled admin API & Prometheus plugin to expose Prometheus metrics as per this document:

https://docs.konghq.com/hub/kong-inc/prometheus/

While inspecting metrics from this endpoint, http://localhost:8001/metrics,

the samples are inconsistent with our tests, Ex: We hit a service endpoint thrice but `kong_http_status{code: 200, service=<our_service>}` is picking only one. So somehow events are lost, are we interpreting the metrics API correctly or does it look like a bug ? 

The same is happening for other metrics too, for example:
`kong_latency_bucket{service=<our_service>,route=<our_route>,type="upstream",le="+Inf"} 1210`. The histogram is also not updated according to service endpoint tests.

### Setup
* EKS cluster
* Kong replicas: 3
* Db-less mode
* Autoscaling: true (max: 12)

**Comment by hbagdi** [comment, marked as answer]
/metrics endpoint outputs the metrics for that specific node. Please hit the same endpoint on other replicas. Total the metrics up and they should add up.

**Reply by narenaryan** [reply]
Got it. Thanks
